gtsfm/retriever/exhaustive_retriever.py

```diff
@@ -14,4 +15,5 @@
 class ExhaustiveRetriever(SequentialRetriever):
     def __init__(self) -> None:
         """Constructor. All frames are considered for matching/co-visibility."""
-        self._max_frame_lookahead = MAX_POSSIBLE_FRAME_LOOKAHEAD
+        super().__init__(max_frame_lookahead=MAX_POSSIBLE_FRAME_LOOKAHEAD)
+        self.matching_regime = ImageMatchingRegime.EXHAUSTIVE
```

Comment:
is this used?

Reply:
if yes, it looks like we are following an underscore prefix for these member variables

Reply:
I see this was updated to `self._matching_regime`, so I'm going to resolve this.